RxTest/Deprecated.swift

```diff
@@ -19,3 +19,43 @@ extension TestScheduler {
         return start(created: created, subscribed: subscribed, disposed: disposed, create: create)
     }
 }
+
+/**
+ These methods are conceptually extensions of `XCTestCase` but because referencing them in closures would
+ require specifying `self.*`, they are made global.
+ */
+//extension XCTestCase {
+    /**
+     Factory method for an `.next` event recorded at a given time with a given value.
+ 
+     - parameter time: Recorded virtual time the `.next` event occurs.
+     - parameter element: Next sequence element.
+     - returns: Recorded event in time.
+     */
+    @available(*, deprecated, renamed: "Recorded.next(_:_:)")
```

Comment:
I don't think we should add `deprecated` attribute at this time because it will create a barrier for updating to next release of RxSwift since this is a pretty big change.

I would like to add this attribute near the end of 4.x lifecycle since this is mostly cosmetic change.

I'm ok with moving then to `Deprecated.swift`.

Reply:
Got it. 

It's better to treat this like `Variable` vs `BehaviorRelay` because a lot of people were using it.